packages/js-draw/src/toolbar/IconProvider.ts

```diff
@@ -765,6 +765,20 @@ export default class IconProvider {
 		);
 	}
 
+	public makePressureSensitivityIcon(): IconElemType {
+		const fill = 'none';
+		const strokeColor = 'var(--icon-color)';
+		return this.makeIconFromPath(
+			`
+    	m 31,83.2 c -50,0 30,-65 -20,-65
+			M 75,17.3 40,59.7 38.2,77.6 55.5,72.4 90.5,30 Z
+    `,
+			fill,
+			strokeColor,
+			'7px',
+		);
+	}
```

Comment:
Since this is a duplicate of `makeStrokeSmoothingIcon`, consider either:
- Having one call the other (may cause issues for custom icon packs), or
- Creating a helper function (similar to `makeRedoIcon`).

Reply:
I put it in as a placeholder, I think the best solution is to put a new icon there that correctly represents the idea of stylus pressure sensitivity. If you could help me find an appropriate icon, that would be great, or if you think that the stroke smoothing icon is good enough, I can create an helper function

Reply:
Most/all of the icons in `toolbar/IconProvider.ts` (the default icon set) are custom. Joplin uses the [material icon pack for js-draw](https://js-draw.web.app/typedoc/modules/_js-draw_material-icons.html). Additional material icons can be found [on Google Fonts](https://developers.google.com/fonts/docs/material_icons).

Reply:
Alright, thanks :) Do you think the "Line Weight" icon is a good candidate?

![image](https://github.com/user-attachments/assets/5be819af-928d-4f34-a7a9-5d9cded5cb01)

If it's okay, I will push this change and all the other ones :D

Reply:
That looks good to me!